benchmark/single-source/SetTests.swift

```diff
@@ -446,6 +959,44 @@ public func run_SetIsDisjointInt(
     }
 }
 
+@inline(never)
+public func run_SetIsDisjointSeqInt(
+    _ a: Set<Int>,
+    _ b: Array<Int>,
+    _ r: Bool,
+    _ n: Int) {
+    for _ in 0 ..< n {
+        let isDisjoint = a.isDisjoint(with: identity(b))
+        CheckResults(isDisjoint == r)
+    }
+}
+
+@inline(never)
+public func run_SetFilterInt50(_ n: Int) {
+  for _ in 0 ..< n {
+    let copy = set.filter { $0 % 2 == 0 }
```

Comment:
To me the name `copy` makes sense only in case when the filter matches all elements… How about this?
```suggestion
    let half = set.filter { $0 % 2 == 0 }
```